gaphor/RAAML/fta/inhibitgate.py

```diff
@@ -42,11 +43,45 @@ def update_shapes(self, event=None):
                     text=lambda: from_package_str(self),
                     style={"font-size": "x-small"},
                 ),
-                style={"padding": (12, 4, 12, 4)},
+                style={
+                    "padding": (55, 4, 0, 4),
+                    "min-height": 100,
+                },
             ),
+            style={"vertical-align": VerticalAlign.BOTTOM},
             draw=draw_inhibit_gate,
         )
 
 
 def draw_inhibit_gate(box, context: DrawContext, bounding_box: Rectangle):
-    draw_border(box, context, bounding_box)
+    cr = context.cairo
```

Comment:
Why not use an SVG image instead?

Reply:
@amolenaar The SVG images are causing the corruption problems for me, so until we resolve that I thought it would be best to just finish the PR by manually drawing them